client/src/containers/edit/EditEducation.js

```diff
@@ -0,0 +1,114 @@
+import React, {Component} from 'react'
+import {
+ MenuItem,
+ TextField,
+ SelectField,
+ DatePicker,
+ RaisedButton
+} from 'material-ui'
+import Save from 'material-ui/svg-icons/content/save'
+import ContentAdd from 'material-ui/svg-icons/content/add'
+import {style} from "./styles";
+import './main.css'
+
+
+class EditEducation extends Component {
+
+
+        render() {
+            return(
+                <div className='fieldsWrapper'>
+                    <div>
+                        <SelectField
+                            floatingLabelText='Select Category of Organization'
+                            underlineStyle={style.textFieldUnderline}
+                            floatingLabelStyle={style.textFloating}
+                            hintStyle={style.textHint}
+                            menuItemStyle={style.textHint}
+                            value={null}
+                        >
+                            <MenuItem primaryText='School'/>
+                            <MenuItem primaryText='Club'/>
+                            <MenuItem primaryText='Program'/>
+                            <MenuItem primaryText='Team'/>
+                        </SelectField>
+                    </div>
+                    <div>
+                        <TextField
```

Comment:
okay, so I see you have a lot of duplicate code here so I'm suggesting we make the TextField and others their own components. Here's an example:
We can create a new folder called widgets and place our new component in there, for TextField it would looks something like this

```
import React from 'react'
import { TextField } from 'material-ui'

import {style} from "./styles"

const StyledTextField = props => {
  const { ...other } = props

  return (
    <div>
      <TextField
          underlineStyle={style.textFieldUnderline}
          floatingLabelStyle={style.textFloating}
          floatingLabelFocusStyle={style.textFloating}
          hintStyle={style.textHint}
          {...other} />
    </div>
  )
}

export default StyledTextField
```

Then we can import that into whatever component we want and pass through additional properties. As you can see, this will decrease duplication in our code and ensure consistency of styles if anyone else is adding these types of fields

Reply:
good suggestion, I went ahead and reorganized the components a bit and placed the styled material ui components in a separate folder.